manage.py

```diff
@@ -334,21 +334,32 @@ def update_sub_schema_reference(schema):
         ])
 
         # Paths that don't appear in the example data at all
-        paths_to_skip = ['forecasts/0/observations/0/value', 'metrics/0/observations/0/value']
+        paths_to_skip = ['forecasts/0/observations/0/value',
+                         'metrics/0/observations/0/value',
+                         'parties/0/beneficialOwners/0',
+                         'parties/0/people/0/address',
+                         'parties/0/people/0/identifier']
 
         # Add examples
         definition["references"] = get_definition_references(schema, defn)
         for ref in definition["references"]:
-            if ref[0] not in schema['definitions'] and '/'.join(ref) not in paths_to_skip:
-                if ref[-1] == '0':
-                    ref.pop(-1)
+            if ref[0] not in schema['definitions']:
```

Comment:
The logic is easier to read as:

```python
            if ref[0] not in schema['definitions'] and not any(p == '/'.join(ref)[:len(p)] for p in paths_to_skip):

Reply:
Ah, that's much nicer! Done in 8e0209c5b89af57b62ab60127837b7f68399e3f9